### ISSUE TITLE
Suggestion about session buttons

### ISSUE THREAD

**Issue by Tuxman2** [heqro/stupid-simple-launcher#6]
Hello heqro,

I've got a new suggestion. Propose an option to display or not the session buttons. I think about people who have small laptop (for example: a 2-in-1-laptop with a 10.x",11.6" or 13.3" screen) and who want to display big icons for the applications on the screen without the session buttons.

What do you think about it ?

About the tooltips of session buttons, is there a way to have it another language that English ?

Thanks.

**Comment by Goldaracena** [comment]
The first one should be easy to get around.

About the languages bit, I am a bit confused because I could have sworn I was using this magic function called i18n that translated stuff whenever it could, but it seems like it has failed this time around. I hope changing languages is easy enough, because I'm going to try debugging this one in my mother tongue. I should have something by this afternoon.

**Comment by Goldaracena** [comment]
Update: the first one is done. I've tried to get the second one done, but it seems like "Lock screen" doesn't want to get automatically translated :/

**Comment by Tuxman2** [comment]
Thanks.

I looked with the other menus available in KDE Store. In the source code of one, I see it: text: i18n("Lock Screen")
Otherwise, perhaps the name is "lock-screen" ?

Regards and Merry Christmas.

**Comment by Goldaracena** [comment]
Yup, I have been trying that exact same i18n string to no avail :/

**Comment by Tuxman2** [comment]
Hello,

very strange that you don't find the way to be able to translate Lock Screen button in another language.

I found it on the net: https://techbase.kde.org/Development/Tutorials/Localization/i18n and https://api.kde.org/frameworks/ki18n/html/prg_guide.html

They talk about i18nc.

Regards.

**Comment by cipricus** [comment]
> an option to display or not the session buttons.

This was implemented. The issue should be closed, right?

**Comment by Goldaracena** [comment]
The issue evolved into the fact that some translations made by Plasma don't actually work, namely the one for lock screen.
